File: app/scratch.py
import pandas as pd
import json

excel_path = "../knowledge-base/RATE CARD JUARA 2026.xlsx"

print(f"Loading {excel_path}...")
df = pd.read_excel(excel_path, sheet_name=0)

print("Excel columns:", df.columns.tolist())
print("\nFirst 10 rows:")
print(df.head(10).to_string())

File: app/scratch.js
import * as xlsx from 'xlsx';

const wb = xlsx.readFile('../knowledge-base/RATE CARD JUARA 2026.xlsx');
console.log("Sheet names:", wb.SheetNames);

const sheetName = wb.SheetNames[0]; // Or maybe there's a sheet called "RATE CARD"
const sheet = wb.Sheets[sheetName];
const data = xlsx.utils.sheet_to_json(sheet, { header: 1 });

for (let i=0; i<10; i++) {
    console.log(`Row ${i}:`, data[i]);
}
